lib/bootloader_setup.pm

```diff
@@ -691,9 +691,13 @@ sub specific_bootmenu_params {
 
 sub remote_install_bootmenu_params {
     my $params = "";
-    if (check_var("VIDEOMODE", "text") || check_var("VIDEOMODE", "ssh-x")) {
+    if (check_var("VIDEOMODE", "vnc")) {
+        $params .= " sshd=1 VNC=1 VNCPassword=$testapi::password "; # trigger vnc installation
```

Comment:
isn't this almost the same as line 702? Since the default case, will be VNC, if `VIDEOMODE` is not `text` or `ssh-x`